### PR TITLE
Session

### PR DESCRIPTION
**Pull request by Ormabe**

### CODE REVIEW

File: backend/routes/user-routes.js [hunk 1]
```diff
@@ -1,54 +1,83 @@
...
-const createNewUser = ((req,res) => {
+const createNewUser = ((req, res) => {
   return Users.findOrCreate({
     where: {
       username: req.body.username,
       email: req.body.email,
-      password: req.body.password
-    }
+      password: req.body.password,
+    },
```
Comment (ChaniceStl) [inline backend/routes/user-routes.js:22]: dont need the comma

File: backend/server/config/passport.js [hunk 2]
```diff
@@ -2,52 +2,66 @@ const passport = require('passport');
...
+//     done(err, user);
+//   });
+// });
 
-passport.deserializeUser(function(id, done) {
-  models.Users.findById(id, function(err, user) {
-    done(err, user);
-  });
+passport.serializeUser((req, user, done) => {
+  console.log("SERIALIZE SESSION ID ====>", req.sessionID)
```
Comment (ChaniceStl) [inline backend/server/config/passport.js:19]: console.log are going to be temp ... (rabiya request)

File: frontend/entryRouter.js [hunk 3]
```diff
@@ -10,7 +10,11 @@ import Feature from './container/main/feature-container.js'
 import ExploreContainer from './container/main/explore-container.js'
 import LocationContainer from './container/main/location.jsx'
 import CultureVault from './cultureVault.js';
+<<<<<<< HEAD
```
Comment (ChaniceStl) [inline frontend/entryRouter.js]: remove the merge conflict message

File: frontend/entryRouter.js [hunk 4]
```diff
@@ -10,7 +10,11 @@ import Feature from './container/main/feature-container.js'
...
+import UserProfile from './container/main/userProfile-container.js';
+=======
 import CreateExperience from './container/experiences/experience-create';
+>>>>>>> master
```
Comment (ChaniceStl) [inline frontend/entryRouter.js]: remove the merge conflict message

File: frontend/entryRouter.js [hunk 5]
```diff
@@ -10,7 +10,11 @@ import Feature from './container/main/feature-container.js'
 import ExploreContainer from './container/main/explore-container.js'
 import LocationContainer from './container/main/location.jsx'
 import CultureVault from './cultureVault.js';
+<<<<<<< HEAD
+import UserProfile from './container/main/userProfile-container.js';
+=======
```
Comment (ChaniceStl) [inline frontend/entryRouter.js]: remove the merge conflict message

File: backend/routes/user-routes.js (hunk 1 above)
Comment (Ormabe) [reply backend/routes/user-routes.js:22]: lint says that indeed it does need trailing comma

**Review by ChaniceStl — APPROVED** [review]

**Pull request merged**